<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple client</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="buttonConnect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="editPort">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="editAddress"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="separatorMessage">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="editLog"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QPushButton" name="buttonSendMessage">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="editMessage"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="Line" name="separatorLog">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>editAddress</tabstop>
  <tabstop>editPort</tabstop>
  <tabstop>buttonConnect</tabstop>
  <tabstop>editMessage</tabstop>
  <tabstop>buttonSendMessage</tabstop>
  <tabstop>editLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
